Silicon/NXP/iMX6Pkg/Drivers/LcdifGop/LcdifGop.inf

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) Microsoft. All rights reserved
```

Comment:
## @file
#
#  Copyright (c) 2019 Microsoft Corporation. All rights reserved.
#
#  This program and the accompanying materials
#  are licensed and made available under the terms and conditions of the BSD License
#  which accompanies this distribution.  The full text of the license may be found at
#  http://opensource.org/licenses/bsd-license.php
#
#  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
#  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
#
##

Reply:
have changed the code according to your feedback